docs/user-guide/install-customization.md

```diff
@@ -149,3 +149,42 @@ curl \
     --data '{"args": ["--append-karg", "nameserver=8.8.8.8", "-n"]}' \
 "http://$ASSISTED_SERVICE_IP:$ASSISTED_SERVICE_PORT/api/assisted-install/v1/clusters/$CLUSTER_ID/hosts/$HOST_ID/installer-args"
 ```
+
+## Adding aditional trust bundle
+Any additional trust bundles need to be added to both the discovery ISO and to the install-config
+### Add additional trust bundle in discovery ISO
+The additional trust bundle must be embeded into discovery ignition override
+```
+request_body=$(mktemp)
+jq -n --arg OVER_RIDE "{\"ignition\": {\"version\": \"3.1.0\"}, \"storage\": {\"files\": [{\"path\": \"/etc/pki/ca-trust/source/anchors/extra_ca.pem\", \"mode\": 420, \"overwrite\": true, \"user\": { \"name\": \"root\"},\"contents\": {\"source\": \"data:text/plain;base64,$(cat ca.pem | base64 -w 0)\"}}]}}" \
```

Comment:
I would use OVERRIDE instead of OVER_RIDE

Reply:
agreed, done